nodejs-build/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+# Run NPM install to download all the NPM packages and webpack to run the
+# babel precompiler.
+- name: npm install
+  shell: npm install --production --no-optional
+  args:
+    chdir: "{{app_src}}/client"
+
+- name: webpack build
+  shell: npm run build:production
```

Comment:
@askldjd would you prefer this be a separate step like this?  If so I'll remove that as a prerequisite from `rake assets:precompile` [here](https://github.com/department-of-veterans-affairs/caseflow/blob/master/config/initializers/react_on_rails.rb#L25).

 Otherwise if I change that line and this runs, we'll double run webpack builds